RELEASE_PROCEDURE.md

```diff
@@ -9,11 +9,13 @@ some Rust versions counts as a breaking change, and requires a major bump.
 
 The release is prepared as follows:
 
+- Ask for a new libc version if, necessary. It usually is.
 - Make a commit with a message like "Release v0.8.3" with the following
   changes:
   - In `CHANGELOG.md`, rename the Unreleased section to the new version
     followed by the date of the release.
   - In `Cargo.toml`, update the version to the new version.
+  - In `Cargo.toml`, change the libc dependency to the latest version.
```

Comment:
add a mention that before committing we should make sure it builds and tests. well I guess this would be picked up by travis when the PR is made.